src/i18n.ts

```diff
@@ -189,6 +189,14 @@ export enum I18N {
   backupFinishCongratulation,
   backupFinishSuccess,
   backupFinishFinish,
+  settingsAccountStyleChoseColor,
+  settingsAccountStyleGenerate,
+  settingsAccountStyleUseStyle,
+  settingsAccountStyleAlreadyUsed,
+  settingsAccountStyleFlat,
+  settingsAccountStyleGradient,
+  settingsAccountStyleCircle,
```

Comment:
put the translations in alphabetical order of 1 letter

Reply:
done